apps/example/src/index.ts

```diff
@@ -72,7 +116,14 @@ new Yab()
 		}),
 	)
 	.use(statics("/public", { assetsDir: "./public" }))
-	.use(router("/api", [UserController]))
+	.use(
+		router("/api", [UserController], {
+			validation: async (schema: any, payload: any) => {
```

Comment:
should specify schema type, value should be unknown

Reply:
change the name to: `customValidation(schema: TSchema, payload: unknown, route: RouteObject)`
user might want to check the path before doing validation, so let them use the whole object instead of schema only